src/util/draws/draw_helper_functions.jl

```diff
@@ -1,7 +1,22 @@
-##############################################
-# Helpful Reference ; https://core.ac.uk/download/pdf/13643059.pdf
-# Converting FOR and MTTR to λ and μ
-##############################################
+"""
+    rate_to_probability(for_gen::Float64, mttr::Int64)
+
+Converts the forced outage rate and mean time to repair to the λ and μ parameters
```

Comment:
Worth it to spell out _lambda_ is **outage_probability** and _mu_ is **recovery_probability**.

Reply:
I see you define it below. Never mind